integration-test/common/conftest.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import os
+import tempfile
+import shutil
+import subprocess
+
+RUN_SCRIPT_FNAME = "run.sh"
+OUTPUT_DIR = "output_dir"
+SWEEP_GENERATOR = "generate_design_sweeps.py"
+
+def is_master(config):
+  """Return true we're on a master node."""
+  return not hasattr(config, 'workerinput')
+
+def pytest_addoption(parser):
```

Comment:
pytest_add_option.

addoption looks like a typo of "adoption"

Reply:
As explained the file-level comment, this is a pytest hook API, so we can't change the name.